### PR TITLE
configuration/hdmi-config.md - clarify which port HDMI 0 is

### PR DESCRIPTION
**Pull request by andrum99**
See #1372

### CODE REVIEW

File: configuration/hdmi-config.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 In the vast majority of cases, simply plugging your HDMI-equipped monitor into the Raspberry Pi using a standard HDMI cable will automatically lead to the Pi using the best resolution the monitor supports. On the Raspberry Pi 4 there are two micro HDMI ports, so you will need either one or two micro-HDMI-to-full-size-HDMI leads or adapters, depending on how many displays you wish to attach. You should connect any HDMI leads before turning on the Raspberry Pi.
```
Comment (lurch) [inline configuration/hdmi-config.md]: Not directly related to this PR, but is it worth squeezing a sentence in here about the Raspberry Pi Zero using a mini HDMI port?  WDYT @JamesH65 ?

Comment (JamesH65) [reply configuration/hdmi-config.md]: Good point - we call out the Pi4, we should also call out the Zero.

**Comment by HughR** [comment]
Thanks for this change.

I don't see a label "HDMI 0" on my board.  How about s/labelled/known as/ ?

**Comment by HughR** [comment]
> Thanks for this change.
> 
> I don't see a label "HDMI 0" on my board. How about s/labelled/known as/ ?

Oops.  Now I do see the label.  Faint and sideways, but there.

**Comment by JamesH65** [comment]
> Thanks for this change.
> 
> I don't see a label "HDMI 0" on my board. How about s/labelled/known as/ ?

Odd, my board has HDMI0 silk screened next to the socket. They all should.

**Comment by JamesH65** [comment]
Ah, OK, crossed in post. Glad you found it, was about to ask for a photo as they all should be clearly marked!

File: configuration/hdmi-config.md (hunk 1 above)
Comment (andrum99) [reply configuration/hdmi-config.md]: I would go the other way - if someone is buying a Pi it should be obvious if there is an adapter needed, from whatever webpage they bought it from. That way the docs stay simple.

Comment (andrum99) [reply configuration/hdmi-config.md]: Although maybe it was me that put the bit about the adaptor for the Pi 4 in to start with lol.

Comment (lurch) [reply configuration/hdmi-config.md]: Already added (and merged) in #1376 :wink:

Comment (andrum99) [reply configuration/hdmi-config.md]: I know.

Comment (JamesH65) [reply configuration/hdmi-config.md]: https://www.youtube.com/watch?v=4LaYn0541ro

**Pull request merged**